inst/include/cpp11/protect.hpp

```diff
@@ -195,23 +198,27 @@ constexpr struct protect safe = {};
 inline void check_user_interrupt() { safe[R_CheckUserInterrupt](); }
 
 template <typename... Args>
-void stop [[noreturn]] (const char* fmt, Args... args) {
-  safe.noreturn(Rf_errorcall)(R_NilValue, fmt, args...);
+void stop [[noreturn]] (const char* fmt_arg, Args... args) {
```

Comment:
Since {fmt} can accept (for example) a std::string without copying it, it'd be better to change these parameter packs to const references
```suggestion
void stop [[noreturn]] (const char* fmt_arg, const Args&... args) {
```